plugin/src/test/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildActionTest.java

```diff
@@ -30,6 +31,25 @@
  */
 @DefaultLocale("en")
 class CoverageBuildActionTest {
+    @SuppressWarnings("unused")
+    private CoverageBuildAction createCoverageBuildActionWithDelta(final Baseline baseline, final Metric metric, final Optional<Fraction> delta) {
```

Comment:
Seems that baseline is not used at all:
```suggestion
    private CoverageBuildAction createCoverageBuildActionWithDelta(final Metric metric, final Optional<Fraction> delta) {
```